src/testUtil/TestCaseRecorder.ts

```diff
@@ -230,10 +233,27 @@ export class TestCaseRecorder {
   }
 
   stop() {
+    this.toggleUserTabSetting(false);
+
     this.active = false;
     this.paused = false;
   }
 
+  private toggleUserTabSetting(shouldSetTabs: boolean) {
```

Comment:
This seems like a weird thing to have a toggle method for, especially since basically the entire method is in the if statement. This probably should just be two different methods.